UI/package.json

```diff
@@ -23,7 +23,7 @@
         "style-loader": "^1.1.3"
     },
     "scripts": {
-        "start": "npm install && react-scripts start",
+        "start": "react-scripts start",
```

Comment:
This was kinda bugging me, cause sometimes I didn't need to reinstall when restarting. How's that feel to you Megan?

Reply:
Agreed